.github/workflows/test.yaml

```diff
@@ -14,10 +14,14 @@ jobs:
         strategy:
             matrix:
                 python-version: [ 3.8, 3.9, "3.10", "3.11", "pypy3.9" ]
-                django: [ 32, 41, 42]
+                django: [ 32, 41, 42, 50]
                 exclude:
                     - python-version: "3.11"
                       django: 32
+                    - python-version: 3.8
```

Comment:
Should the pypy3.9 also be added to the exclusion list for django 5?

[https://docs.djangoproject.com/en/5.0/releases/5.0/#python-compatibility](https://docs.djangoproject.com/en/5.0/releases/5.0/#python-compatibility)

Reply:
@BradleyKirton Nice catch; had 3.9 in exclusions but missed pypy3.9. Added now!